app/assets/javascripts/controllers/header/header_controller.js

```diff
@@ -20,13 +21,24 @@ function HeaderCtrl($scope, eventNotifications, $timeout) {
   var refresh = function() {
     $timeout(function() {
       vm.newNotifications = eventNotifications.state().unreadNotifications;
+      updateTooltip();
     });
   };
 
   var destroy = function() {
     eventNotifications.unregisterObserverCallback(refresh);
   };
 
+  function updateTooltip() {
+    var unreadNotificationCount = 0;
+    if (angular.isArray(vm.notificationGroups)) {
+      angular.forEach(vm.notificationGroups, function(group) {
+        unreadNotificationCount += group.unreadCount;
+      });
+    }
+    vm.notificationsIndicatorTooltip = __(unreadNotificationCount + " unread notifications");
```

Comment:
..aah, wait, there is a better way..

`miqFormatNotification("%{count} unread notifications", { count: { text: unreadNotificationCount } })`